web/crux/src/decorators/jwt-user.decorator.ts

```diff
@@ -0,0 +1,8 @@
+import { createParamDecorator, ExecutionContext } from '@nestjs/common'
+
+const JWTUser = createParamDecorator((_: any, context: ExecutionContext) => {
```

Comment:
any? 🌷

Reply:
Yes because the request body type is unknown. I could maybe do some generic magic but the request body is not needed anyways.